website/docs/jellyfish/api/governance.md

```diff
@@ -68,3 +68,18 @@ interface ProposalInfo {
   payoutAddress: string
 }
  ```
+
+## createVoc
+
+Creates a Vote of Confidence.
+
+```ts title="client.governance.createVoc()"
+interface governance {
+  createVoc (title: string, utxos: UTXO[]): Promise<string>
```

Comment:
Should be this right? `createVoc (title: string, utxos: UTXO[] = []): Promise<string>`

Reply:
yes, good catch !